pyrightconfig.typeshed.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I'd prefer it if this config file were called something like `pyrightconfig.scripts_and_tests.json` — `pyrightconfig.typeshed.json` doesn't feel very descriptive. The whole project's typeshed! 😀

Reply:
Fair point! Will rename